.rubocop.yml

```diff
@@ -1,17 +1,25 @@
+inherit_from: .rubocop_todo.yml
+
```

Comment:
Should we be inheriting from the mas-standards rubocop here?

Reply:
A fair point @malcolmbaig @tomas-stefano. Added it and updated the `.rubocop_todo.yml` file as we might need to address rubocop better in another instance.